solidity/random-beacon/contracts/RandomBeaconGovernable.sol

```diff
@@ -0,0 +1,693 @@
+//   ▓▓▓▓▓▓▀▀▓▓▓▓▓▓▄       ▐▓▓▓▓▓▓▀▀▀▀         ▓▓▓▓▓▓▀▀▀▀         ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▀
+//   ▓▓▓▓▓▓   ▀▓▓▓▓▓▓▄     ▐▓▓▓▓▓▓     ▓▓▓▓▓   ▓▓▓▓▓▓     ▓▓▓▓▓   ▐▓▓▓▓▓▌
+// ▓▓▓▓▓▓▓▓▓▓ █▓▓▓▓▓▓▓▓▓ ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  ▓▓▓▓▓▓▓▓▓▓
+// ▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓ ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  ▓▓▓▓▓▓▓▓▓▓
+//
+//                           Trust math, not hardware.
+
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.5;
+
+import "./GovernanceUtils.sol";
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/// @title Random Beacon Governable
+/// @notice This is a base contract that holds governable parameters for Random
+///        Beacon V2 as well as their update logic
+contract RandomBeaconGovernable is Ownable {
+    /// @notice The time delay that needs to pass between initializing and
+    ///         finalizing update of any governable parameter in this contract.
+    uint256 public constant GOVERNANCE_DELAY = 7 days;
+
+    /// @notice Relay request fee in T
+    uint256 public relayRequestFee;
+    uint256 public newRelayRequestFee;
+    uint256 public relayRequestFeeChangeInitiated;
+
+    /// @notice Reward for submitting DKG result
+    uint256 public dkgResultSubmissionReward;
+    uint256 public newDkgResultSubmissionReward;
+    uint256 public dkgResultSubmissionRewardChangeInitiated;
+
+    /// @notice Reward for unlocking the sortition pool if DKG timed out
+    uint256 public sortitionPoolUnlockingReward;
+    uint256 public newSortitionPoolUnlockingReward;
+    uint256 public sortitionPoolUnlockingRewardChangeInitiated;
+
+    /// @notice Slashing amount for not submitting relay entry
+    uint256 public relayEntrySubmissionFailureSlashingAmount;
+    uint256 public newRelayEntrySubmissionFailureSlashingAmount;
+    uint256 public relayEntrySubmissionFailureSlashingAmountChangeInitiated;
+
+    /// @notice Slashing amount for submitting malicious DKG result
+    uint256 public maliciousDkgResultSlashingAmount;
+    uint256 public newMaliciousDkgResultSlashingAmount;
+    uint256 public maliciousDkgResultSlashingAmountChangeInitiated;
+
+    /// @notice The number of blocks for which a DKG result can be challenged
+    uint256 public dkgResultChallengePeriodLength;
+    uint256 public newDkgResultChallengePeriodLength;
+    uint256 public dkgResultChallengePeriodLengthChangeInitiated;
+
+    /// @notice The number of blocks for a member to become eligible to submit
+    ///         relay entry
+    uint256 public relayEntrySubmissionEligibilityDelay;
+    uint256 public newRelayEntrySubmissionEligibilityDelay;
+    uint256 public relayEntrySubmissionEligibilityDelayChangeInitiated;
+
+    /// @notice The number of blocks for a member to become eligible to submit
+    ///         DKG result
+    uint256 public dkgSubmissionEligibilityDelay;
+    uint256 public newDkgSubmissionEligibilityDelay;
+    uint256 public dkgSubmissionEligibilityDelayChangeInitiated;
+
+    /// @notice Hard timeout for a relay entry
```

Comment:
`in blocks`

Reply:
Done in https://github.com/keep-network/keep-core/pull/2613/commits/29f18f6ce0f8b89b14a1f21f8c801aa9fbea1ace.